### PR TITLE
Add link to profile in automated emails

### LINKED ISSUE #833: add link to profile in automated emails
"You are receiving this email because you have an active account on Trustroots."

"an active account" should link to the user account

### PR DESCRIPTION
**Pull request by laazee**
#### Proposed Changes

*Tried to insert link to user profile page... did I break it?

#### Testing Instructions

*

Fixes #833

### CODE REVIEW

File: modules/core/server/services/email.server.service.js [hunk 1]
```diff
@@ -475,6 +476,13 @@ exports.addEmailBaseTemplateParams = function (params) {
     content: 'email-support'
   });
 
+  params.profileUrl = analyticsHandler.appendUTMParams(params.urlProfilePlainText, {
+    source: 'transactional-email',
+    medium: 'email',
+    campaign: params.utmCampaign || 'transactional-email',
+    content: 'email-support'
```
[the remark this answers is not in the document]
Comment (guaka) [reply modules/core/server/services/email.server.service.js]: What did you mean with "this"?

[the remark this answers is not in the document]
Comment (simison) [reply modules/core/server/services/email.server.service.js]: Sorry, I meant to change:

`content: 'email-support'`

to:

`content: 'email-profile'`

**Comment by guaka** [comment]
awesome 👬 pair programming with @mrkvon here on the 🛋 couch

File: modules/core/server/services/email.server.service.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (guaka) [reply modules/core/server/services/email.server.service.js]: cool, did this in [e59c1f6], together with some DRYing

File: modules/core/server/services/email.server.service.js [hunk 2]
```diff
@@ -452,6 +452,7 @@ exports.addEmailBaseTemplateParams = function (params) {
   var baseUrl = (config.https ? 'https' : 'http') + '://' + config.domain;
 
   params.urlSupportPlainText = baseUrl + '/support';
+  params.urlProfilePlainText = baseUrl + '/profile/' + user.username;
```
[the remark this answers is not in the document]
Comment (guaka) [reply modules/core/server/services/email.server.service.js]: `if` and slightly diff approach in e59c1f6

**Comment by simison** [comment]
I'll test it the first change I'll get. 👍

It touches a lot of critical stuff so I want to give it a good spin.

**Comment by simison** [comment]
Thanks for handling this, everyone!

We can still add the `username` to quite a few more emails in that file:

https://github.com/Trustroots/trustroots/blob/e59c1f63014baefefbd6eddeb63010b386fde5cb/modules/core/server/services/email.server.service.js

...but that's good for a follow-up PR.

**Pull request merged**